packages/common/infra/src/modules/doc/services/docs.ts

```diff
@@ -68,4 +70,45 @@ export class DocsService extends Service {
     }
     return docRecord;
   }
+
+  async addLinkedDoc(targetDocId: string, linkedDocId: string) {
+    const { doc, release } = this.open(targetDocId);
+    doc.setPriorityLoad(10);
```

Comment:
this is a update behavior, why load priority set?

Reply:
We need to wait for doc loading ready for the following writing. If we don't set the priority, it may take a long time.